src/backend/distributed/citus--5.2-1--5.2-2.sql

```diff
@@ -0,0 +1,7 @@
+/* citus--5.2-1--5.2-2.sql */
+CREATE FUNCTION cluster_metadata_snapshot()
```

Comment:
We only use this UDF for testing right now. Maybe we can omit this here and create the function in the relevant regression test like we do with other functions from the test/ folder.